src/helpers/action.ts

```diff
@@ -53,7 +53,7 @@ export function debugInfoForReference(reference: any): string {
 
 function debugName(obj: any) {
   let objType = typeof obj;
-  if (obj == null) {
```

Comment:
Same as above, I think this should be updated to `reference === null || reference === undefined`.